tldr/cross_file_calls/builder.py

```
@@ -0,0 +1,192 @@
+        respect_gitignore=respect_gitignore,
+        parallel=parallel,
+        max_workers=max_workers,
+    )
+    
+    # Resolve cross-file calls
+    _resolve_cross_file_calls(call_graph, verbose)
+    
+    return call_graph
+
+
+def _resolve_cross_file_calls(call_graph: ProjectCallGraph, verbose: bool = False) -> None:
```

Ruff (0.14.11)

Boolean-typed positional argument in function definition

(FBT001)